kbase-extension/scss/partials/_narrative.scss

```diff
@@ -1987,7 +1987,7 @@ div#notebook_panel { /* FIXME Jupyter css */
     display: inline;
     margin: 3px 13px 3px 3px;
     color: rgb(46 97 141);
-    font: normal 400 12pt/15px $typeface-alt-page-text;
+    font: normal 400 16px/1 $typeface-alt-page-text;
```

Comment:
The line height here was smaller than the font size, which could lead to some visual overlapping ==> I set the line height to 1 (same as font size).